tests/python/gaia-ui-tests/gaiatest/tests/functional/music/test_music_change_rating.py

```diff
@@ -45,4 +45,3 @@ def test_music_change_rating(self):
         #change the rating to 1
         player_view.tap_star(1)
         self.assertEquals(player_view.star_rating, 1)
-        
```

Comment:
I believe the no new line at end of file error is wrong. In the editor the is a new line.
What I changed here is that I removed some white space from the line.